spec/httparty/request_spec.rb

```diff
@@ -664,6 +664,18 @@
           expect(response.parsed_response).to eq({"hash" => {"foo" => "bar"}})
         end
 
+        it "redirects to an absolute path if a 300 contains a relative path in the location header" do
+          redirect = stub_response '', 300
+          redirect['location'] = '../bar/baz'
+          ok = stub_response('<hash><bar>baz</bar></hash>', 200)
+          allow(@http).to receive(:request).and_return(redirect, ok)
+          response = @request.perform
```

Comment:
This needs to be surrounded by newlines from above and below to visually separate the test into 3 sections _Arrange, Act, Assert_

Reply:
Done